docs/docs/market-details.md

```diff
@@ -0,0 +1,105 @@
+  - [How many Manifold team members in the Bahamas will test positive for COVID?](https://manifold.markets/Sinclair/how-many-manifold-team-members-in-t)
+
+## Free-Response Markets: Betting & Payouts
+
+- Markets are structured around a list of answers, any of which can be bet on.
+- Anyone can add answers to a market as long as they stake some amount of M$ on it. Traders can place a bet on any answer and receive shares in the outcome in return.
+- When a user places a bet, their M$ goes into the market's pool and they receive a certain amount of shares of the selected answer.
+- When the market is resolved, you will be paid out according to your shares. If the creator resolves to answer #1, the entire pool is divided up amongst the users who bet on answer #1 proportional to their shares.
+- The creator of each market is responsible for resolving each market. They can resolve to any single answer, or even multiple answers.
+  - Resolving to multiple answers allows the creator to choose a percentage for each selected answer (or distribute equally). The payout for any answer is taken from the amount of the total pool allocated to that answer.
+    - For example, let's take a free-response market with many answers. The pool for this market is $500, and you own 100 out of 500 total shares of answer #1.
+    - If the creator resolves to answer #1 only, you will receive `M$500 * (100 / 500) = M$100`.
+    - If the creator resolves 50% to answer #1 and 50% to answer #2, you will receive `(M$500 * 50%) * (100 / 500) = M$50`.
+    - Note that your payout is dependent on the total number of shares, and thus may decrease if more people buy shares in that answer.
+
+# Fees
+
+- Manifold charges fees on each trade. They are automatically calculated and baked into the number of shares you receive when you place a bet.
+  - Our CPMM fee schedule is currently: `10% * (1 - post-bet probability) * bet amount`
+    - Note that all current binary markets use this fee schedule.
+    - The post-bet probability is what the market probability would be after your bet if there were no fees.
+    - Example:
+      - If you bet M$100 on NO and the resulting probability without fees would be 10%, then you pay `M$100 * 10% * 10% = M$1.0`.
+      - If you bet M$100 on YES and the resulting probability without fees would be 50%, then you pay `M$100 * 10% * 50% = M$5.0`.
+    - 100% of this fee is used to provide a commission to the market creator, which is paid out after the market is resolved.
+  - Our DPM fee schedule is currently: `5% * (1 - post-bet probability) * bet amount`
+    - Note that all free-response markets use this fee schedule. The calculation for this is the same as above.
+    - 4% is used to provide a commission to the market creator, which is paid out after the market is resolved. 1% is "burnt" to prevent inflation.
+  - No fees are levied on sales.
```

Comment:
If you have existing shares in a binary market and buy shares on the opposite side, that is equivalent to selling your shares and you do not pay fees